Makefile

```diff
@@ -182,6 +182,7 @@ apis:
 	@$(ROOT)/scripts/format-json.sh applications/crossbar/priv/couchdb/schemas/*.json
 	@ERL_LIBS=deps/:core/:applications/ $(ROOT)/scripts/generate-api-endpoints.escript
 	@$(ROOT)/scripts/format-json.sh applications/crossbar/priv/api/swagger.json
+	@ERL_LIBS=deps/:core/:applications/ $(ROOT)/scripts/generate-fs-headers-hrl.escript
```

Comment:
So when fs_event_filters.hrl is actually used (this PR builds it but does not use it!), we should move that line to whatever app that's using this HRL. Otherwise builds will not be checking in the proper version of this HRL file, over time. Reminder: our build system does not and has never supported tracking HRL-ERLs build dependencies.